components/TextFields/src/Theming/MDCTextInputControllerOutlined+MaterialTheming.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <MaterialComponents/MaterialTextFields.h>
+#import <MaterialComponentsBeta/MaterialContainerScheme.h>
+
+/**
+ This category is used to style MDCTextInputControllerOutlined instances to a specific Material
+ style which can be found within the [Material
+ Guidelines](https://material.io/design/components/text-fields.html).
+ */
+@interface MDCTextInputControllerOutlined (MaterialTheming)
+
+/**
+ Applies a container scheme to this instance.
```

Comment:
_Nit:_ Applies a Material theme to this instance

Reply:
I just stuck with the language in the MDCTextInputControllerFilled theming extension

Reply:
@andrewoverton Please correct this in a follow-up PR. If you see other errors in Text Fields, please correct them as well.  We shouldn't propagate mistakes to new code when we catch them.

Reply:
@romoore I didn't think there was anything wrong with it :/ But if you feel strongly I'm happy to go back and change it.

Should I have it specify "id<MDCContainerScheming>" rather than "container scheme"? Or do you like Cody's suggestion?

Reply:
@codeman7  is correct - the method applies a Material theme to the receiver.  It's not applying a scheme object (regardless of its type) to the receiver.  The scheme is a necessary parameter to the behavior of performing the theming.

Reply:
Sounds good :)

Reply:
I think it's also worth mentioning that non-*Tip:* comments should be addressed to the reviewer's satisfaction before merging. In this case it was a Nit, but a worthwhile change to help  the reader understand the purpose of the API.